recipes/moleculetda/LICENSE

```diff
@@ -0,0 +1,42 @@
+molecule-tda Copyright (c) 2021, The Regents of the University of California,
```

Comment:
The license is already included in the source tarball so you don't need to (and shouldn't) include it in the recipe. Please go ahead and remove.

Reply:
Thanks, removed the license!